src/Components/SimpleModal/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./SimpleModal";
```

Comment:
Maybe it is better to name the component `Modal`? It is a drop-in replacement for `react-native`'s Modal, so it is better to name it exactly the same.